BCStudentSoftwareDevTeam/celts
# Remove all occurences of endDate

## Issue Description

Fixes issue #1348
- End date is current a field in our database for events but it is never needed nor necessary. Remove it from the database and wherever else it appears.

## Changes
- Deleted endDate from the event model
- Deleted all unnecessary references to endDate in the tests.
- **It should be noted that endDate is still necessary in some instances for event preprocessing for recurring events, but it should never make it to the database.**

## Testing
- Checkout `1348_remove_end_data`
- Source setup
- Reset database
- **Run tests**
- Go through event functionality like creating recurring and non recurring events, viewing events, etc. and verify nothing is breaking or working incorrectly.
- Look through code and tests.
